.dockerignore

```diff
@@ -0,0 +1,4 @@
+.drone.yml
+deploy
+node_modules
```

Comment:
This is a very good point. I should remind myself to check we're doing it everywhere else too.